src/react-components/ui-root.js

```diff
@@ -1020,7 +1023,7 @@ class UIRoot extends Component {
                     e.target.blur();
                   }
                 }}
-                placeholder="Send to room..."
+                placeholder="Send message to others..."
```

Comment:
Hmm... this copy feels awkward to me but am not sure if I can think of something better (that also fits on small screens). 
How about "Message the room..."?